### PR TITLE
Page title

### PR DESCRIPTION
**Pull request by alicjamakowska**

### CODE REVIEW

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (michudzik) [inline .vs/ProjectSettings.json]: Remove .vs from version control + already pushed commits.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.title-background {
+    background-color: #4E81E0;
```
Comment (michudzik) [inline main.css]: Try to use CSS variables in order to store particular colours. As time goes it gets easier to maintain when those things are stored in this way. (No colour mismatches)

https://www.w3schools.com/css/css3_variables.asp

File: main.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    padding-top: 3px;
+    width: 175px;
+    height: 30px;
+    border-radius: .5rem;
+    background-color: #4873C5;
+}
+
+.h1 {
+    font-size: 2rem;
+}
```
Comment (michudzik) [inline main.css]: New line at the end of the file.

File: scripts/countdown.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    const days = Math.floor(t / (1000 * 60 * 60 * 24));
+    const hours = Math.floor((t % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+    const minutes = Math.floor((t % (1000 * 60 * 60)) / (1000 * 60));
+    document.getElementById("countdown").innerHTML = days + "dni "
+        + hours + "godz. " + minutes + "min.";
+    if (t < 0) {
+        clearInterval(x);
+        document.getElementById("countdown").innerHTML = "Wydarzenie trwa";
+    }
+}, 1000);
```
Comment (michudzik) [inline scripts/countdown.js]: New line at the end of the file.

File: scripts/countdown.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const deadline = new Date("Apr 14, 2020 10:00:00").getTime();
+const x = setInterval(function () {
+    const now = new Date().getTime();
+    const t = deadline - now;
+    const days = Math.floor(t / (1000 * 60 * 60 * 24));
```
Comment (michudzik) [inline scripts/countdown.js]: Oh boi. Can we extract  some of the calculations which are frequent to separate functions?

File: main.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+.counter {
+    padding-top: 3px;
+    width: 175px;
+    height: 30px;
+    border-radius: .5rem;
+    background-color: #4873C5;
+}
+
+.h1 {
```
Comment (michudzik) [inline main.css]: is a class named `h1`?

File: main.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
+.title-background {
+    background-color: #4E81E0;
+}
+
+.page-title {
+    font-size: 2rem;
+    line-height: 1.2;
```
Comment (michudzik) [inline main.css]: Try to write units next to a CSS value.

File: main.css [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+.counter {
+    padding-top: 3px;
+    width: 175px;
+    height: 30px;
+    border-radius: .5rem;
```
Comment (michudzik) [inline main.css]: Personally I would avoid using rems - they are hard to manage (still this is my personal opinion :) )

File: scripts/countdown.js [hunk 9]
```diff
@@ -0,0 +1,14 @@
+const deadline = new Date("Apr 14, 2020 10:00:00").getTime();
```
Comment (michudzik) [inline scripts/countdown.js]: If we are to have multiple `.js` files in this project i would advise to keep content of each and every of them enclosed in an object - the scopes won't override each other by accident and variables which are stored in different files may be named the same. So for example:

```
const Counter = {
  deadline: new Date [....],
  x: () => {
    setInterval(() => { [...] }, 1000)
  }
}
Counter.x();
```

File: main.css (hunk 6 above)
Comment (alicjamakowska) [reply main.css]: I had to overwrite the h1 font height to match paragraphs according to Michał Kidawa

**Pull request merged**